src/main/kotlin/by/architecture_map/belarus/entity/Architect.kt

```diff
@@ -0,0 +1,39 @@
+package by.architecture_map.belarus.entity
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.JoinTable
+import jakarta.persistence.ManyToMany
+import jakarta.persistence.OneToOne
+
+@Entity
+data class Architect(
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        var id: Int? = null,
```

Comment:
better to create a common class with the id fields and all the configurations, and reuse it